src/pages/how-to/retrieve.mdx

```diff
@@ -15,7 +15,7 @@ When retrieving any data, you'll be using the content CID of the upload (prefixe
 
 ## Using an IPFS HTTP gateway
 
-You can easily fetch any data stored using Storacha using an IPFS HTTP gateway. Because IPFS is a peer-to-peer, decentralized network, you can use any public HTTP gateway to fetch your data. In this guide, we'll use the gateway at w3s.link (which is optimized for data stored with Storacha), but you can see more worldwide gateways on the [IPFS Public Gateway Checker](https://ipfs.github.io/public-gateway-checker/).
+You can easily fetch any data stored using Storacha using an IPFS HTTP gateway. Because IPFS is a peer-to-peer, decentralized network, you can use any public HTTP gateway to fetch your data. In this guide, we'll use the gateway at w3s.link (which is optimized for data stored with Storacha), but you can see more worldwide gateways on the [IPFS Public Gateway Checker](https://ipfs.github.io/public-gateway-checker/). To see the full possibile passable parameters, check out [Trustless Gateway Specifications](https://specs.ipfs.tech/http-gateways/trustless-gateway/#http-request).
```

Comment:
```suggestion
You can easily fetch any data stored using Storacha using an IPFS HTTP gateway. Because IPFS is a peer-to-peer, decentralized network, you can use any public HTTP gateway to fetch your data. In this guide, we'll use the gateway at w3s.link (which is optimized for data stored with Storacha), but you can see more worldwide gateways on the [IPFS Public Gateway Checker](https://ipfs.github.io/public-gateway-checker/). To see the full set of parameters and options supported by our gateway, please check out the [Trustless Gateway Specification](https://specs.ipfs.tech/http-gateways/trustless-gateway/#http-request).
```